crates/ignore/src/default_types.rs

```diff
@@ -190,6 +190,8 @@ pub const DEFAULT_TYPES: &[(&str, &[&str])] = &[
     ("rdoc", &["*.rdoc"]),
     ("readme", &["README*", "*README"]),
     ("red", &["*.r", "*.red", "*.reds"]),
+    ("reasonml", &["*.re", "*.rei"]),
```

Comment:
This should come before `red` and after `readme`.

Reply:
Indeed. Fixed!